bigquery/samples/export_data_to_cloud_storage.py

```diff
@@ -30,6 +31,8 @@ def export_table(service, cloud_storage_path,
         cloud_storage_path: fully qualified
         path to a Google Cloud Storage location,
         e.g. gs://mybucket/myfolder/
+        export_format: format to export in;
+        "CSV", "NEWLINE_DELIMITED_JSON", or "AVRO".
```

Comment:
nit
I think it is normal to indent the continuation in the argument section. Also capitalize the first letter of each section and add period instead of comma at the end of the section.
```
def export_table(service, cloud_storage_path, projectId, datasetId, tableId,
                 num_retries=5):
    """
    Starts an export job

    Args:
        service: Initialized and authorized bigquery google-api-client object.
        cloud_storage_path: Fully qualified path to a Google Cloud Storage
            location (e.g. gs://mybucket/myfolder/).

    Returns: An extract job resource representing the job. See
        https://cloud.google.com/bigquery/docs/reference/v2/jobs
    """
    pass
```